src/Compilers/Core/MSBuildTask/BuildClient.cs

```diff
@@ -133,13 +139,18 @@ private static int HandleResponse(BuildResponse response)
             IList<string> arguments,
             CancellationToken cancellationToken,
             string keepAlive = null,
-            string libEnvVariable = null)
+            string libEnvVariable = null,
+            string gacPath = null)
```

Comment:
The name 'gacPath' is a bit misleading. It's the path to use if the EXE is running from the GAC.